tests/test_edgeql_data_migration.py

```diff
@@ -140,7 +141,7 @@ async def fast_forward_describe_migration(
                         interpolations[var_name] = var_value
 
                 for stmt in mig['proposed']['statements']:
-                    curddl = stmt['text']
```

Comment:
We don't want to have to an add `unescape_string` here. This is testing exposed behavior; if we update this, we'd need to update our CLI's migration flows, and we'd break any user using this.